data-prepper-plugins/otel-trace-group-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/oteltracegroup/OTelTraceGroupPrepper.java

```diff
@@ -36,18 +36,23 @@
 @DataPrepperPlugin(name = "otel_trace_group_prepper", type = PluginType.PREPPER)
 public class OTelTraceGroupPrepper extends AbstractPrepper<Record<String>, Record<String>> {
 
+    public static final String SPANS_MISSING_TRACE_GROUP_INFO = "spansMissingTraceGroupInfo";
```

Comment:
I should've noted this earlier, but do we want more metrics than just failures? Some that come to mind are:
1. Records in which are missing trace group
2. Records out which were successfully fixed
3. Records out which were unsuccessfully fixed (which is what the highlighted line is)

Would also suggest renaming SPANS_MISSING_TRACE_GROUP_INFO to something more indicative that a failure occurred - I could see spansMissingTraceGroupInfo being interpreted as either scenarios 1 or 3 above.

Reply:
Finer resolution sounds reasonable. Renaming should be necessary for that.

Reply:
@wrijeff Updated the metrics and also the README section. Might better receive another look.